README.md

```diff
@@ -273,6 +275,16 @@ This is useful when new docs content is published, usually corresponding to new
 
 In case you want to save the current version and convert it into a historical one, you can run `npm run make-historical -- -p <pkg-name>` beforehand.
 
+If the version is not for the latest stable minor release series, then add `--historical` to the arguments. For example, use `--historical` if the latest stable release is 0.45.\* but you're generating docs for the patch release 0.44.3.
```

Comment:
This belongs in the `Generate patch releases` docs because it's only relevant for that.

If you're worried that people won't realize they need to read that section, then I would add to the top of this current section something like this:

> If you're generating a patch release, also see the below subsection for additional steps.